examples/java/src/main/java/org/apache/beam/examples/DebuggingWordCount.java

```diff
@@ -50,9 +50,8 @@
  *
  * <p>New Concepts:
  * <pre>
- *   1. Logging to Cloud Logging
- *   2. Controlling worker log levels
- *   3. Creating a custom aggregator
+ *   1. Logging using SLF4J
+ *   3. Creating a custom aggregator (runners have varying levels of support)
```

Comment:
Concept 2 intentionally left blank... better to not even let on about its existence. Done.